[Bug] SingleLineAfterImportsFixer - no meaningful after open semicolon

SpacePossum opened the pull request:
Intergration tests for (all most) empty files.

Conflicts:
	Symfony/CS/Fixer/Symfony/NativeFunctionCasingFixer.php
	Symfony/CS/Tests/Fixer/Contrib/HeaderCommentFixerTest.php

SpacePossum commented:
replaces https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1961

SpacePossum commented:
green so ready?

keradus commented on Symfony/CS/Tests/IntegrationTest.php:
```diff
@@ -35,4 +37,99 @@ protected static function getTempFile()
+     *
+     * @dataProvider provideBasicSources
+     */
+    private function doTest($source, array $whiteListFixerNames = array())
```
Please, could you provide some information for that method?
on 2.x we have it: https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/master/src/Test/AbstractIntegrationTestCase.php#L152

but I don't think they are combatible, am I right ?

SpacePossum commented:
Updated, I've moved the test to its own file.
Please note that the test relies on some fixer names, unfortunately the following have been renamed on master (so please be aware when merging upstream)
```
eof_ending -> single_blank_line_at_eof
short_echo_tag -> no_short_echo_tag
php_closing_tag -> no_closing_tag
```

keradus commented on Symfony/CS/Tests/BasicFileFixingTest.php:
```diff
@@ -0,0 +1,119 @@
+use Symfony\CS\Fixer;
+
+/**
+ * Test the behavior of all fixers on some basic PHP files.
```
I can't get the purpose of this test

SpacePossum replied:
Maybe it no longer has a use, but given the bugs found and fixed in this PR and this one; https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1978 the test had a purpose.
I'll delete it

keradus replied:
I just does too much at same time. And previously it was IntegrationTest right ?

keradus commented:
@SpacePossum why closing this one ?

SpacePossum commented:
Sorry forgot the ref.: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1989
I was making conflicts in my own PR's...

keradus commented:
Great, thanks !
